blank-canvas-blocks/experimental-theme.json

```diff
@@ -224,6 +224,11 @@
 							"hoverBackground": "var(--wp--custom--color--background)"
 						}
 					}
+				},
+				"table": {
+					"typography": {
+						"fontSize": "var(--wp--preset--font-size--tiny)"
```

Comment:
Can this declaration live directly under "styles" -> "core/table" now that we've decoupled block supports and theme.json for supported style attributes?

Reply:
No I don't think so because in this case it is being mapped to the caption, not the table itself. Maybe the format of the theme.json should better represent that?

Reply:
Yea, images and such keep that in a sub-object.  Makes sense to do that here as well I think.

Reply:
Good call. Done in 6b8b6dd1